website/docs/concepts/router.md

```diff
@@ -242,6 +398,92 @@ In its simplest form this is just a `HashMap` containing string pairs.
 `location.query` is used to obtain the query parameters.
 It uses `serde` to deserialize the parameters from query string in the URL.
 
+## Nested Router
```

Comment:
@hamza1311 
I know #1853 will improve how this can be implemented. But no harm to add some missing docs first right? :D

Reply:
Yes. This won't be improving in this release so the current way doing it should be in the docs